lib/fwupd_service.dart

```diff
@@ -158,4 +161,20 @@ class FwupdService {
     log.debug('verifyUpdate $device');
     return _fwupd.verifyUpdate(device.id);
   }
+
+  Future<void> reboot() async {
+    final manager = SessionManager();
```

Comment:
could you allow injecting a `SessionManager` instance through the constructor similarly to the fwupd client, please?